fuzz/fuzz_targets/int_set_op_processor.rs

```diff
@@ -721,6 +721,64 @@ where
     }
 }
 
+/* ### Iter Excluded Ranges ### */
+
+struct IterExcludedRangesOp;
+
+impl IterExcludedRangesOp {
+    fn parse_args<T>() -> Option<Box<dyn Operation<T>>>
+    where
+        T: SetMember,
+    {
+        Some(Box::new(Self))
+    }
+}
+
+impl<T> Operation<T> for IterExcludedRangesOp
+where
+    T: SetMember,
+{
+    fn operate(&self, input: Input<T>, _: Input<T>) {
+        let mut btree_ranges: Vec<RangeInclusive<T>> = vec![];
+        let mut cur_range: Option<RangeInclusive<T>> = None;
+
+        let mut inverted = BTreeSet::<T>::new();
+        for v in T::ordered_values() {
+            let v = T::create(v).unwrap();
+            if !input.btree_set.contains(&v) {
+                inverted.insert(v);
+            }
+        }
```

Comment:
just  golf but I'd write this whole thing as,

```rust
let inverted: BTreeSet<_> = T::order_values()
    .map(|v| T::create(v).unwrap())
    .filter(|val| !input.btree_set.contains(&v))
    .collect();

Reply:
looks good, thanks